handel/Cargo.toml

```diff
@@ -14,7 +14,7 @@ log = { package = "tracing", version = "0.1", features = ["log"] }
 parking_lot = "0.12"
 rand = "0.8"
 thiserror = "1.0"
-tokio = { version = "1.26", features = ["rt", "time"] }
+tokio = { version = "1.26", features = ["rt", "macros", "time"] }
```

Comment:
This shouldn't be needed since it was redefined in `dev-dependencies` with this feature